diff --git a/dist/runtime/npm/debug.mjs b/dist/runtime/npm/debug.mjs
index 62fa13e3a5e96ee835dfe7363e7451e987ef11bc..85b923f5f5597519486b5a11604af6dbfadb0a0d 100644
--- a/dist/runtime/npm/debug.mjs
+++ b/dist/runtime/npm/debug.mjs
@@ -1,7 +1,17 @@
 function createDebug(namespace) {
 	return Object.assign((...args) => {
 		const env = globalThis.process?.env.DEBUG;
-		if (!env || env !== "*" && !env.startsWith(namespace)) return;
+		if (!env) return;
+		if (env !== "*") {
+		  const patterns = env.split(",").map((n) => n.trim());
+		  const isMatch = patterns.some((pattern) => {
+		    if (pattern.endsWith("*")) {
+		      return namespace.startsWith(pattern.slice(0, -1));
+		    }
+		    return pattern === namespace;
+		  });
+		  if (!isMatch) return;
+		}
 		console.debug(...args);
 	}, {
 		color: "#000000",
